x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/AuthenticationService.java

```diff
@@ -691,19 +698,9 @@ private void authenticateToken(AuthenticationToken token) {
         }
     }
 
-    abstract static class AuditableRequest {
-
-        final AuditTrail auditTrail;
-        final AuthenticationFailureHandler failureHandler;
-        final ThreadContext threadContext;
-
-        AuditableRequest(AuditTrail auditTrail, AuthenticationFailureHandler failureHandler, ThreadContext threadContext) {
-            this.auditTrail = auditTrail;
-            this.failureHandler = failureHandler;
-            this.threadContext = threadContext;
-        }
+    abstract static class AuthenticatedRequest {
```

Comment:
I don't think we should call this `AuthenticatedRequest` because at this point it is not authenticated; it is the request that we are attempting to authenticate. Unfortunately all the names I am thinking of are not very good.